source/Plugins/GoogleAnalyticsV4/README

```diff
@@ -0,0 +1,421 @@
+/*
+  Copyright 2014 Google Inc. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+Google Analytics Plugin for Unity (beta)
+Copyright (c) 2014 Google Inc. All rights reserved.
+
+The Google Analytics Plugin for Unity allows game developers to easily implement Google Analytics in their Unity games on all platforms, without having to write separate implementations. Note that this is a beta and as such may contains bugs or other issues. Please report them through the Github issue tracker or submit a pull request. The plugin comes with no guarantees.
+
+Unity is a trademark of Unity Technologies. This project is not in any way endorsed or supervised by Unity Technologies.
+
+iOS is a trademark of Apple, Inc.
+
+Google Analytics V3 Plugin Installation & Set-up:
```

Comment:
This still talks about V3 (from the V4 bundle), probably worth updating in a future change.